pkg/apiserver/registry/clusterpedia/collectionresources/rest.go

```diff
@@ -109,7 +109,7 @@ func (s *REST) Get(ctx context.Context, name string, _ *metav1.GetOptions) (runt
 
 	storage, ok := s.storages[name]
 	if !ok {
-		return nil, errors.New("")
+		return nil, fmt.Errorf("CollectionResourceStorage failed to get key: %v\n", name)
```

Comment:
Thanks for fixing this legacy issue from alpha.

I think this might be better, otherwise it would be `500 Internal Server Error`.
```go
                #  import apierrors "k8s.io/apimachinery/pkg/api/errors"

                return nil, apierrors.NewNotFound(
                        schema.GroupResource{Group: internal.GroupName, Resource: "collectionresources"},
                        name,
                )
```

Reply:
Thanks for your suggestion !  I have changed